### DISCUSSION TITLE
Unklarheiten & offene Fragen

### DISCUSSION THREAD

**Discussion by vladislabv** [vladislabv/etl-pipeline-big-data#1] [Ideas]
Wie wir in dem letzten Meeting angedeutet haben, sammeln wir alles, was nicht klar ist und dann den Prof./Dozent diesbezüglich fragen.

Bitte schickt eure Vorschläge/Fragen hier.

**Comment by PatrickUlbrich** [comment]
Offene oder zu klärende Punkte, soweit ich es noch weiß:
1. Bestätigung, dass die Aufgabe "Group 2" auch wirklich MongoDB ist
2. "Abnahme" des Schemas unter nachfolgendem Link, dass es so grundsätzlich passend ist: https://dbdocs.io/stasenko_vladislav/ETL-Pipeline-MongoDB-Scratch?table=measure_values&schema=public&view=table_structure 
3. Was genau ist mit "make suggestion on optimizations and transformations of the data" gemeint - reicht mehr oder weniger ein reines Empfangen und Speichern der JSON aus, oder was soll noch erfolgen?

**Reply by Hendimetz** [reply]
Was muss von uns alles gespeichert werden? Auch die measurements?

**Reply by Hendimetz** [reply]
Soll nachher nur nach bestimmten Typen der Sensoren gesucht werden oder immer alle Werte?

**Comment by vladislabv** [comment]
Ich schreibe mal die inhaltlichen Fragen mit Rückblick auf API:

1. model.GatewayConfig
  a. api_timeout ist die Zeit in Sekunden, die zwischen jeder Abfrage eingehalten werden muss oder erst wenn einige Einschränkungen gestoßen wurden, z.B. max_allowed_clients überschritten.
  b. poll_interval Was bedeutet der Feld überhaupt?
  c. Hat version Feld eine praktische Bedeutung für die API?

2. model.Gateway
  a. Hat network_segment Feld eine praktische Bedeutung für die API?
  b. Kann man sich auf die ID Feld verlassen, sprich die ID ist vergeben durch Hardware oder was auch immer oder wird dynamisch zugewiesen?
  c. Kann es bei einem Gateway die IP4 Adresse sich ändern?
  
3. model.Tag 
  a. Warum hat einen Tag, laut der API, immer eine leere Menge von Sensoren?
  b. Kann man einen Tag einzigartig mit einem Wert aus name Feld definieren?
  c. Kann es bei einem Tag die IP6 Adresse sich ändern (also im Laufe der Zeit oder einem Neustart von dem Tag)?
